addons/point_of_sale/static/src/app/screens/partner_list/partner_editor/partner_editor.js

```diff
@@ -31,6 +31,15 @@ export class PartnerDetailsEdit extends Component {
             vat: partner.vat || "",
             property_product_pricelist: this.setDefaultPricelist(partner),
         });
+        this.translatedItems = {
+            'Street': _t('Street'),
+            'City': _t('City'),
+            'Zip': _t('Zip'),
+            'Email': _t('Email'),
+            'Phone': _t('Phone'),
+            'Mobile': _t('Mobile'),
+            'Barcode': _t('Barcode')
+        };
```

Comment:
Nice work! Can you just change the name to the `translatedItems` variable (to something like `partnerDetailsFields`) and put a comment saying that you put the different terms in the js because if it was in the XML, it would not be translated ?

Reply:
Yes I will do that, thank you for the feedback.